src/test/lib/operators/print_test.cpp

```diff
@@ -148,7 +148,7 @@ TEST_F(OperatorsPrintTest, GetColumnWidths) {
 }
 
 TEST_F(OperatorsPrintTest, OperatorName) {
-  auto pr = std::make_shared<hyrise::Print>(_gt, PrintFlags::None, output);
+  auto pr = std::make_shared<Print>(_gt, PrintFlags::None, output);
```

Comment:
`const auto print`

Reply:
Since everything else looks good to me, I will address changing this line in #2490 to avoid re-running the full CI again here.